docs-gen/content/rule_set/basics.md

```diff
@@ -4,16 +4,16 @@ date: 2019-08-04T13:05:11+02:00
 weight: 1
 ---
 ## Specification format
-A domain specification is written as a flat YAML list, where each signal and
-branch is a self-contained YAML list element.
 
-The YAML list is a single file, called a *vspec* file.
+The Vehicle Signal Specification domain specification consist of *vspec* files.
+*vspec* files are YAML lists following the rule set defined for VSS.
```

Comment:
I think we are not "lists" anymore. "vspec files are YAML files" or "vspec files contain YAML defintions following..."